### ISSUE TITLE
Syntax highlight breaks display of our config presets

### ISSUE THREAD

**Issue by HonkingGoose** [renovatebot/renovatebot.github.io#99]
**What browser are you using?**

Firefox 94.0.2 (64 bit) on Windows 11

**Describe the bug**

The config JSON is displayed too big.

Affected page example: https://docs.renovatebot.com/presets-config/

![bad-display-config-preset](https://user-images.githubusercontent.com/34918129/144246816-a6e64971-00d9-4339-a6bc-d5eb0d4d62d5.png)

**Steps to reproduce**

1. Go to https://docs.renovatebot.com/presets-config/
1. See display of JSON config in too big font size

**Additional context**

The JSON displayed properly before PR #97. I'm not sure _why_ it breaks though.

**Comment by HonkingGoose** [comment]
I've just tested this on Gitpod by doing a `git revert` on the commit that introduced the syntax highlighting.

This is what we had before:
![after-revert](https://user-images.githubusercontent.com/34918129/144248931-9a18a4d2-b949-4e05-87e8-ebbc1a831899.png)

**Comment by viceice** [comment]
looks like we need some more newlines after the markdown headers

**Comment by HonkingGoose** [comment]
Do you mean that we should change this code?

https://github.com/renovatebot/renovatebot.github.io/blob/61fa1588eb889742bb24573cfe67fd91a848a051/bin/generate-presets.js#L1-L81

**Comment by viceice** [comment]
probabbly yes

**Comment by HonkingGoose** [comment]
The config presets page also breaks when I try adding admonitions [^docs], with the following options in the `mkdocs.yml` file:

```yaml
markdown_extensions:
  - admonition
  - pymdownx.details
  - pymdownx.superfences
```

[^docs]: https://squidfunk.github.io/mkdocs-material/reference/admonitions/

**Comment by HonkingGoose** [comment]
I think the code we need to change is in one of these files here maybe?

https://github.com/renovatebot/renovate/tree/main/tools

**Comment by viceice** [comment]
Yes, i think we need to check the prepared markdown files for issues. I think there are some missing newlines

**Comment by HonkingGoose** [comment]
I think this issue should be moved to the main renovate repository as well, as the file is also on the main repo. 😄
